package.json

```diff
@@ -149,7 +149,7 @@
     "@fortawesome/react-fontawesome": "^0.1.12",
     "@reduxjs/toolkit": "^1.5.0",
     "assert": "^1.4.1",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
```

Comment:
ahh I ran into this too. 
Wasn't there another one ? 
ansi-regex
Introduced through cypress@7.3.0,
Fixed in
ansi-regex@6.0.1, @5.0.1

Reply:
Yes, but it didn't stop my PR so I didn't look into it.